static/app/components/profiling/flamegraphZoomViewMinimap.tsx

```diff
@@ -50,7 +50,9 @@ function FlamegraphZoomViewMinimap({
         },
       });
 
-      if (previousRenderer?.flamegraph.name === renderer.flamegraph.name) {
+      if (
+        previousRenderer?.flamegraph.profile.name === renderer.flamegraph.profile.name
+      ) {
```

Comment:
Should this condition be synced with the condition used in `flamegraphZoomView`?

Reply:
Yeah. I think we should just start thinking about what it would take to just pass one flamegraphRenderer around 🤔 Idk if we need to necessarily initialize two of them, we should just be able to render one into whatever view we want